src/mscorlib/shared/System/DllNotFoundException.cs

```diff
@@ -16,6 +16,8 @@
 
 namespace System
 {
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
Since we have this string in tons of places, could we put it in a single constant somewhere, to avoid potential issues of copy-paste errors, typos, etc.?

Reply:
That's a good suggestion. But I won't do it in this PR as this is intended to be ported to the release branch. When I replace all the assembly qualified names I would also other touch other files.